pkg/utils/utils.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021 Nordix Foundation.
```

Comment:
Utils modules are dangerous - they usually end up as a junkyard for bunch of unrelated functions, making it difficult to navigate the project.

Could you please keeps this under `utils/cache.go` ?

Reply:
yes, done